Dockerfile

```diff
@@ -1,7 +1,12 @@
-FROM node:14-alpine as build
+FROM node:14-alpine as base
 WORKDIR /usr/app
 RUN apk update && apk add --no-cache git g++ make python && rm -rf /var/cache/apk/*
 
+FROM base as development
+RUN apk add --no-cache bash
+RUN yarn global add ts-node typescript
```

Comment:
Tried locally and this code ran always even when targeting production. Could you find a different arrangement so the production build doesn't have to do extra work exclusive to development?

Reply:
i will check it out and see how to optimize.

Reply:
hey @dapplion , i did a rebuild with no cache for just production, it actually didn't build the development target. image attached.
![image](https://user-images.githubusercontent.com/76567250/111909890-0c810e80-8a85-11eb-8632-6124dcc3aa7f.png)

Reply:
@dapplion if you are using docker build ., then it would be building all targets, if you are using docker build . --target production then it won't. just trying to guess why you saw all the targets building :)